web/src/hooks/queries/useUser.ts

```diff
@@ -57,7 +57,7 @@ export const useUserQuery = (address?: Address, where?: Dispute_Filter) => {
   const { graphqlBatcher } = useGraphqlBatcher();
 
   return useQuery<UserQuery | UserDisputeFilterQuery>({
-    queryKey: [`userQuery${address?.toLowerCase()}`],
+    queryKey: [`userQuery${address?.toLowerCase()}${where}`],
```

Comment:
I think this way of adding keys is more idiomatic to react-query 🤔 
```suggestion
    queryKey: ["userQuery", address?.toLowerCase(), JSON.stringify(where)],
```